modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -603,6 +603,12 @@ public String toOperationId(String operationId) {
             operationId = "call_" + operationId;
         }
 
+        // operationId starts with a number
+        if (operationId.matches("^\\d.*")) {
+            LOGGER.warn(operationId + " (starting with a number) cannot be used as method name. Renamed to " + camelize(sanitizeName("call_" + operationId)));
+            operationId = camelize("call_" + operationId);
```

Comment:
I think "operation" as a prefix makes more sense. "Call" suggests an operation is being performed, which may not always be the case (for example if the templates are modified to create infrastructural services rather than endpoint invokers).

Reply:
also you are logging the sanitized name here, and then don't sanitize it when assigning it to the variable operationId

Reply:
Nice catch. This should probably assign, then log the assigned value to avoid issues in the future.

Reply:
@jimschubert yup, agree there may be a better choice of word other than "call_". My goal is to make the output compiles without issues due to method names starting with numbers. Users need to rename the operationId if they want complete control.

Reply:
@stkrwork good catch. Let me revise that.